test/jdk/javax/swing/JFileChooser/FileChooserSymLinkTest.java

```diff
@@ -39,10 +29,21 @@
 
 import javax.swing.JFileChooser;
 import javax.swing.JFrame;
+import javax.swing.JOptionPane;
 import javax.swing.SwingUtilities;
 import javax.swing.WindowConstants;
 
-public class FileChooserSymLinkTest{
+/** @test
+  * @bug 8281966
+  * @key headful
+  * @library /java/awt/regtesthelpers
+  * @build PassFailJFrame
+  * @summary Test to check if the absolute path of Symbolic Link folder
+  *          is valid on ValueChanged property listener.
+  * @run main/manual FileChooserSymLinkTest
+*/
+
```

Comment:
I'd rather remove this blank line so that the comment above is attached to the class declaration.